### DISCUSSION TITLE
changing the default branch name

### DISCUSSION THREAD

**Discussion by iacore** [david-vanderson/dvui#57] [General]
What do you think of changing the default branch name of this project to `main`? You can also set it globally, I think.

**Comment by david-vanderson** [comment]
Good idea - I have done so on github.  When I did, it offered the following recipe for how to change it locally for each person:
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a

**Reply by iacore** [reply]
I had to also do

```
git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
```
since I made a shallow clone at first.
